_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy.cpp

```diff
@@ -996,6 +996,9 @@ void Legacy::InitInternal(const FeatureBlocks& /*blocks*/, TPushII Push)
         auto dflts = GetRTDefaults(strg);
         SetVPS(dflts, Glob::VPS::GetOrConstruct(strg));
 
+        auto sts = dflts.base.GetVPS(dflts, Glob::VPS::Get(strg));
```

Comment:
You'll get an error here in case when Interlace feature not included into HEVCe impl.
Need to push base/default function logic inside Legacy feature.
Move code from SetVPS to hevcehw_base_legacy_defaults.cpp - GetDefault::VPS; then add PUSH_DEFAULT(VPS) to GetDefault::Push(Defaults& df) similar to SPS/PPS:
https://github.com/Intel-Media-SDK/MediaSDK/blob/338d0b573a01ad0cc443a0b98c78e23264fad37e/_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy_defaults.cpp#L1818-L1823
https://github.com/Intel-Media-SDK/MediaSDK/blob/338d0b573a01ad0cc443a0b98c78e23264fad37e/_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy_defaults.cpp#L2188

Reply:
Got it.